File: src/main/java/com/architecture_map/belarus/repository/ArchitecturalStyleRepository.java
package com.architecture_map.belarus.repository;

import com.architecture_map.belarus.entity.ArchitecturalStyle;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ArchitecturalStyleRepository extends JpaRepository<ArchitecturalStyle, Integer> {

    boolean existsByName(String name);
}

File: src/main/kotlin/by/architecturemap/belarus/repository/jpa/ArchitecturalStyleRepository.kt
package by.architecturemap.belarus.repository.jpa

import by.architecturemap.belarus.entity.ArchitecturalStyle
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ArchitecturalStyleRepository : JpaRepository<ArchitecturalStyle, Int>
